src/Transformer/FractalTransformer.php

```diff
@@ -76,7 +76,9 @@ public function transform($response, $transformer, Binding $binding, Request $re
         }
 
         if ($response instanceof EloquentCollection) {
-            $response->load($this->fractal->getRequestedIncludes());
+            $requestedIncludes = $this->fractal->getRequestedIncludes();
+            $eagerLoads = $this->getEagerLoads($transformer, $requestedIncludes);
+            $response->load($eagerLoads);
```

Comment:
It gets a bit more complicated than that though. True eager loading changes `n + 1` into `1`, which is what Eloquent offers with `User::with('posts')->get();`. When you first do `User::all()` and then use `$users->load('posts)`, you are really executing `1 + 1` SQL queries, hence it's called ['lazy eager loading'](http://laravel.com/docs/4.2/eloquent#eager-loading). So maybe in cases where you really want just `1` query to get executed, you'd want to disable the `->load()`.